static/src/views/components/atoms/Label.jsx

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import styled from 'styled-components'
+
+const Label = styled.label`
+  display: block;
+  font-size: 16px;
```

Comment:
@jannydiamond we should add a base font-size to our gobal styles (lets include it into our theme and just read the fontsize from the theme inside our global styles)